src/odbc_result.cpp

```diff
@@ -416,7 +416,7 @@ double odbc_result::as_double(nanodbc::timestamp const& ts) {
 
 double odbc_result::as_double(nanodbc::date const& dt) {
   using namespace cctz;
-  auto sec = convert(civil_day(dt.year, dt.month, dt.day), c_->timezone());
+  auto sec = convert(civil_day(dt.year, dt.month, dt.day), c_->timezone_out());
```

Comment:
Is this still correct after #295?

Reply:
I think I misunderstood how the double value stored in POSIXct class. I thought the value would be varied depending on the timezone but it's not. It's a fixed value compared to UTC 1970-1-1 00:00:00. 

> Class "POSIXct" represents the (signed) number of seconds since the beginning of 1970 (in the UTC time zone) as a numeric vector. 

So what should be done is: after fetching from DB, we just assign the timezone attributes that the user wants. There's no need to change the value of the underlying.

In conclusion, this line is not used at all (the function is about `nanodbc::date` and will never be called for a datetime value) and we actually don't need `c_->timezone_out()`. Only `c_->timezone_out_str()` will be used. 

I will file the correction later.